### PR TITLE
Team page

### PR DESCRIPTION
**Pull request by Isaac-Best**
Team page, mobile and desktop view

- LinkedIn icons link to respective profiles
- Advisory committee links -> no info yet on where to link users currently just links to teams page again

### CODE REVIEW

File: src/app/team/page.tsx [hunk 1]
```diff
@@ -1,5 +1,100 @@
...
+                <img src="/team/icon-linkedin.svg" alt="LinkedIn company logo" className="h-6 w-6" />
+              </Link>
+            </p>
+          </div>
+        </ContentContainer>
+        <ContentContainer>
+          <div>
+            <p className="text-base lg:text-2xl font-medium">Legal Advisors</p>
+            <p className="text-base lg:text-lg">Oroma Womeodu, Esq.</p>
+            <p className="text-base lg:text-lg">Jade Monkouopm</p>
```
Comment (Andrew-Dame) [inline src/app/team/page.tsx:39]: Screen reader users will be pleased if content visually represented as a list is represented as such using `ul`/`li`.

**Review by Andrew-Dame — CHANGES_REQUESTED** [review]

File: src/app/team/page.tsx [hunk 2]
```diff
@@ -1,5 +1,100 @@
...
+              <p className="text-base lg:text-lg">Michael Shetzer
+                <Link href="/team" className="inline-block ml-3">
+                  <img src="/team/icon-link.svg" alt="Two chains interlinked" className="h-6 w-6" />
+                </Link>
+              </p>
+            </div>
+          </ContentContainer>
+        </div>
+        <div className="basis-1/2">
+          <img src="/team/illustration-01.svg" alt="3 people forming a huddle"/>
```
Comment (Andrew-Dame) [inline src/app/team/page.tsx:95]: I believe it's grammatically correct to say `three` here instead of `3`.

File: src/app/team/page.tsx (hunk 1 above)
Comment (Isaac-Best) [reply src/app/team/page.tsx]: Thanks for all the feedback, I believe I have implemented the fixes for all of it.

**Review by RuochenRockyWang — APPROVED** [review]
Looks good

**Review by Andrew-Dame — APPROVED** [review]

**Pull request merged**